Makefile

```diff
@@ -7,6 +7,7 @@ VERSION ?=latest
 
 IMG ?= quay.io/openshift/origin-cloud-event-proxy:$(VERSION)
 CONSUMER_IMG ?= quay.io/redhat-cne/cloud-event-consumer:$(VERSION)
+T_TYPE?=AMQ
```

Comment:
How does this env variable get passed into the configuration? and is there a need to set it AMQ by default?